src/components/InstancesList/InstancesList.js

```diff
@@ -348,6 +349,25 @@ class InstancesView extends React.Component {
     this.setState({ showErrorModal: false });
   };
 
+  toggleRowSelection = rowId => {
+    this.setState(({ selectedRows }) => {
+      const isRowSelected = Boolean(selectedRows[rowId]);
+      const newSelectedRows = { ...selectedRows };
+
+      if (isRowSelected) {
+        delete newSelectedRows[rowId];
```

Comment:
Do we need to delete that? I think it might look nicer if we drop the `if` statement by just setting row status to `false`.
 `newSelectedRows[rowId] = isRowSelected;`

Reply:
In this case we will collect items which are not really needed. So at the end in order to get the selected items it would just enough to get object keys without any filtering logic.

Reply:
Makes sense.